Uplift/Controllers/OnboardingGymsViewController.swift

```diff
@@ -126,7 +126,7 @@ class OnboardingGymsViewController: UIViewController, UITableViewDelegate, UITab
         gymsTableView.snp.makeConstraints { make in
             make.top.equalTo(titleLabel.snp.bottom).offset(28)
             make.leading.trailing.equalToSuperview().inset(gymSidePadding)
-            make.bottom.equalTo(nextButton.snp.top).offset(14)
+            make.bottom.equalTo(nextButton.snp.top).offset(-14)
```

Comment:
i think you might be able to use `inset(14)` here?

Reply:
I'm getting identical results with inset and offset for that constraint in particular

Reply:
same as in `offset(-14)` and `inset(14)` does the same thing?

Reply:
yea- This is with inset(14):
<img width="334" alt="image" src="https://user-images.githubusercontent.com/19177346/64544737-80215780-d2f5-11e9-8ca3-8c0da2f62093.png">

This is with offset(14):
<img width="328" alt="image" src="https://user-images.githubusercontent.com/19177346/64544841-b4951380-d2f5-11e9-9919-27b3193c2795.png">

And an example with inset(-14):
<img width="318" alt="image" src="https://user-images.githubusercontent.com/19177346/64545046-0fc70600-d2f6-11e9-9943-9b64aaedb6e3.png">

Reply:
could you ask design what they think of the table being cut off before you scroll and maybe how to better indicate that this is scrollable? i feel like visually this might not be the best solution